src/components/DataTable/DataTable.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+  DataTable,
+  DataTableBody,
+  DataTableEmpty,
+  DataTableHeader,
+  DataTableRow,
+} from "."
+
+export default {
+  title: "Components/Data Table",
+}
+
+const Template = {
+  children: "Data Table",
+}
+
+export const Empty = () => (
+  <DataTable>
+    <DataTableEmpty />
```

Comment:
We need to support colspan on `DataTableEmpty` to make it render correctly on tables with headers.

We should have two separate stories:

- `Empty`
- `Empty with headers`

Reply:
Shouldn't it be whoever's using this component, the responsability of passing the right el ?